tools/ci_build/github/linux/docker/inference/x64/default/cpu/scripts/install_deps.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
This file is identical with the one in aarch64 folder.

Reply:
do they need to be kept consistent? is there a way to avoid duplication?

Reply:
Use a script to generate them.